src/main/java/Task.java

```diff
@@ -0,0 +1,37 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description, boolean isDone) {
+        this.description = description;
+        this.isDone = isDone;
+    }
+
+    public void markTaskAsDone() {
+        this.isDone = true;
+    }
+
```

Comment:
Perhaps look into creating short JavaDoc comments for public classes and methods to aid in readability?